src/commands/new.ts

```diff
@@ -47,124 +56,33 @@ const command: GluegunCommand = {
-
-  // Install Pods
-  const installPodsSpinner = print.spin('Installing pods...');
-  await system.run(`cd ${projectName}/ios && pod install --repo-update && cd ../`);
-  installPodsSpinner.stop();
-
-  print.success(`${projectName} has been created!`);
-};
-
-const createJetProject = async (toolbox: Toolbox, opts: Options) => {
-  const { print, filesystem, template } = toolbox;
-  const { projectName, projectDesc } = opts;
-  const { log, printTask, cmd, postInstallInstructions } = interfaceHelpers(toolbox);
+const createProject = async (projectName: string, template: RepoLocation, toolbox: Toolbox) => {
+  const { print, filesystem } = toolbox;
+  const { log, postInstallInstructions } = interfaceHelpers(toolbox);
 
-  let task;
+  // TODO: remove this whe "Propeller" is ready
+  if (template === REPO_URL_TEMPLATE_PROPELLER) return print.info('Coming soon!');
 
-  task = printTask('🦾 Activating robot assemblers...');
-  await cmd(`git clone ${REPO_URL_TEMPLATE_JET} ${projectName}`);
-  task.stop();
+  await cloneTemplateRepo(projectName, template, toolbox);
 
   if (!filesystem.exists(`${projectName}/package.json`))
     throw new Error(`Something went wrong. ${projectName}/package.json file not found.`);
 
   log(`changing directory to \`${projectName}\``);
   process.chdir(projectName);
 
-  task = printTask('💼 Supplying secret codenames...');
-  await cmd(`npx react-native-rename ${projectName}`);
-  task.stop();
-
-  task = printTask('🪐 Folding dependencies into 4D space...');
-  await cmd(`yarn install --ignore-scripts`);
-  await cmd(`pod repo update`);
-  await cmd(`cd ios && pod install && cd ..`);
-  task.stop();
-
-  task = printTask('🛠  Transmogrifying starship components...');
-  // update package.json
-  const editPkgJson = `npx json -I -f package.json -e`;
-  await cmd(`${editPkgJson} 'this.name="${decamelize(projectName, { separator: '-' })}"'`);
-  await cmd(`${editPkgJson} 'this.version="${DEFAULT_PROJECT_VERSION}"'`);
-  if (projectDesc) await cmd(`${editPkgJson} 'this.description="${projectDesc}"'`);
-  // generate code of conduct
-  await cmd(`npx covgen ${AIRSHIP_EMAIL}`);
-  // create README
-  await template.generate({
-    template: 'README.md.ejs',
-    target: 'README.md',
-    props: { projectName, projectDesc },
-  });
-  // apply prettier so that app is already formatted correctly
-  await cmd(`npx prettier --ignore-path .gitignore --write \"app/**/*.+(tsx|jsx|ts|js)\"`);
-  task.stop();
-
-  task = printTask('🧠 Uploading consciousness to Git...');
-  await cmd(`rm -rf .git`);
-  await cmd(`git init`);
-  await cmd(`git add --all`);
-  await cmd(`git commit -m "Initial commit ⚡️ Automated by Airfoil™ 🚀"`);
-  // rename 'master' branch as 'main'
-  await cmd(`git branch -M main`);
-  task.stop();
-
-  print.success(`${red(projectName)} successfully Airfoil'ed. 🚀`);
+  await renameReactNativeProject(projectName, toolbox);
+  await installDependencies(toolbox);
+  await initializeProjectInfo(projectName, toolbox);
```

Comment:
Can we make `toolbox` the default first param for all script actions? That way params are more consistent from one function to the next.